ogr/ogrsf_frmts/gml/gmlhandler.cpp

```diff
@@ -1275,7 +1275,8 @@ OGRErr GMLHandler::startElementFeatureAttribute(const char *pszName,
             return startElementGeometry(pszName, nLenName, attr);
         }
     }
-    else if (nLenName == 9 && strcmp(pszName, "boundedBy") == 0)
+    else if (nLenName == 9 && strcmp(pszName, "boundedBy") == 0 &&
+             (eAppSchemaType == APPSCHEMA_CITYGML || m_poReader->UseBBOX()))
```

Comment:
Maybe document that the open option `USE_BBOX=NO` has no effect for CityGML ?

Is this because of https://github.com/OSGeo/gdal/issues/2084?

Reply:
> Maybe document that the open option `USE_BBOX=NO` has no effect for CityGML ?

I've added this comment in the code:
             // We ignore the UseBBOX() flag for CityGML, since CityGML
             // has elements like bldg:boundedBy, which are not a simple
             // rectangular bbox. This is needed to read properly
             // autotest/ogr/data/gml/citygml_lod2_713_5322.xml
             // (this is a workaround of not being namespace aware)

So I don't think this deserves a user doc addition

> Is this because of https://github.com/OSGeo/gdal/issues/2084? > 

no, because of the fix for qgis/QGIS#51647